build/sign.proj

```diff
@@ -15,23 +15,14 @@
 
   <UsingTask TaskName="ESRPSignTask" AssemblyFile="$(CISignRepo)\tools\sdkbuildtools\tasks\MS.Az.Sdk.OnPremise.Build.Tasks.dll" />
 
-  <Target Name="net452">
-    <Message Importance="high" Text="Sign: net452 started" />
-    <ItemGroup>
-      <UnsignedFiles Include="$(OutputDir)net452\Microsoft.Azure.Commands*.dll" />
-      <UnsignedFiles Include="$(OutputDir)net452\Microsoft.WindowsAzure.Commands*.dll" />
-      <UnsignedFiles Include="$(OutputDir)net452\Microsoft.Azure.PowerShell*.dll" Exclude="$(OutputDir)net452\Microsoft.Azure.PowerShell*.Test.dll" />
-    </ItemGroup>
-  </Target>
-
   <Target Name="netstandard20">
     <Message Importance="high" Text="Sign: netstandard20 started" />
     <ItemGroup>
       <UnsignedFiles Include="$(OutputDir)netstandard2.0\Microsoft.Azure.PowerShell*.dll" Exclude="$(OutputDir)netstandard2.0\Microsoft.Azure.PowerShell*.Test.dll" />
     </ItemGroup>
   </Target>
 
-  <Target Name="SignFiles" AfterTargets="net452;netstandard20;All" Condition="'$(DisableSigning)' != 'true'">
+  <Target Name="SignFiles" AfterTargets="netstandard20;All" Condition="'$(DisableSigning)' != 'true'">
```

Comment:
This complex `SignFiles` logic was only done so that net452 and netstandard20 ItemGroups could be combined together and batch signed. Now that net452 is removed, we can just move this logic into the netstandard20 target, remove is target and the `BeforeAll` target, and update `All` to only depend on `netstandard20`.